apps/studio/src/lib/db/clients/sqlite.js

```diff
@@ -14,14 +14,20 @@ import { ClientError } from '../client';
 const log = rawLog.scope('sqlite')
 const logger = () => log
 
-const knex = knexlib({ client: 'better-sqlite3'})
+const knex = knexlib({ client: 'better-sqlite3',
+  // silence the "sqlite does not support inserting default values" warnings on every insert
+  useNullAsDefault: true,
+})
 
 const sqliteErrors = {
   CANCELED: 'SQLITE_INTERRUPT',
 };
 
 const PD = SqliteData
 
+// Fix (part 1 of 3) Issue #1399 - int64s not displaying properly
+// Fixes "TypeError: Do not know how to serialize a BigInt"
+BigInt.prototype.toJSON = function() { return this.toString() }
```

Comment:
This modifies BigInts for the whole app if I'm reading this right?

If so we probably need to put this somewhere more obvious. I think somewhere like `src/common/initializers/big_int_initializer.ts`. You'll then also need to include and run this in `main.ts` and also in the test runners.

Reply:
It didn't work in the `.ts` file, which turned out to be A Good Thing because it reminded me that comments on the StackOverflow question that I got it from had this exchange:

> This works with pure JavaScript, but not TypeScript. Compiling with tsc gives error `TS2339: Property 'toJSON' does not exist on type 'BigInt'.  
> [Bruce](https://stackoverflow.com/users/4361237/bruce)

> There's a TypeScript solution in that link if you keep reading:
> - https://github.com/GoogleChromeLabs/jsbi/issues/30#issuecomment-1006088574  
> 
> [Adam Spiers](https://stackoverflow.com/users/179332/adam-spiers)

So I added the more complete JS-and-TS solution to the new initializer.  Fixing the test runner now

Reply:
Hmm, not sure why that build failed.  Looks like an Apple-y thing: 

>  skipped macOS application code signing  reason=cannot find valid "Developer ID Application" identity or custom non-Apple code signing certificate, it could cause some undefined behaviour, e.g. macOS localized description not visible, see https://electron.build/code-signing allIdentities=     0 identities found
>                                                 Valid identities only
> Skipping notarization: Authentication environment variables for notarization are missing. Either APPLE_ID and APPLE_ID_PASSWORD, or API_KEY_ID and API_KEY_ISSUER_ID must be defined.
> Error: Exit code: ENOENT. spawn /usr/bin/python ENOENT

But, actually, it seems like this should be moved into that new `/initializers/` file, so i'll psh that and see if it builds then

Reply:
Ok despite the (apparently unrelated) build failures, I think this is good to go.  Sorry for all the back and forth (e.g., dialects/index.ts is no longer even modified).  I'm happy to create a new branch and PR that is cleaner to commit to master, if you'd like! let me know